JavaTraining/BillsBurgers/src/Burger.java

```diff
@@ -0,0 +1,111 @@
+public class Burger {
+
+    protected String name;
+    protected double burgerPrice;
+    protected String breadType;
+    protected boolean meat;
+    protected boolean lettuce;
+    protected double lettucePrice = 0.60;
+    protected boolean cheese;
+    protected double cheesePrice = 1;
+    protected boolean onions;
+    protected double onionsPrice = 0.60;
+    protected boolean tomato;
+    protected double tomatoPrice = 0.75;
+
+    public Burger(String name, String breadType, double burgerPrice) {
+        this.name = name;
+        this.breadType = breadType;
+        this.burgerPrice = burgerPrice;
+        this.meat = true;
+    }
+
+    public Burger(String name, int burgerPrice) {
+        this.name = name;
+        this.burgerPrice = burgerPrice;
+        this.meat = true;
+
+    }
+    public void setLettuce(boolean lettuce) {
```

Comment:
If these are burger add ons, could we perhaps think of a way to apply "Tell, don't ask" principle, for even better encapsulation?